okhttp/src/main/java/okhttp3/internal/platform/AndroidPlatform.java

```diff
@@ -431,7 +431,16 @@ public int hashCode() {
   }
 
   @Override public SSLContext getSSLContext() {
-    if (Build.VERSION.SDK_INT >= 16 && Build.VERSION.SDK_INT < 22) {
+    boolean tryTls12 = true;
+    try {
+      tryTls12 = (Build.VERSION.SDK_INT >= 16 && Build.VERSION.SDK_INT < 22);
+    } catch (NoClassDefFoundError e) {
+      // Not a real Android runtime; probably RoboVM or MoE
+      // Try to load TLS 1.2 explicitly.
+      tryTls12 = true;
```

Comment:
This line can be removed

Reply:
Ops, right. I actually just removed the initial assignment to make the intention more explicit.